client/package.json

```diff
@@ -3,7 +3,7 @@
   "scripts": {
     "dev": "next",
     "test": "jest --coverage",
-    "lint": "eslint --ext .js,.ts .",
+    "lint": "npx eslint components/**",
```

Comment:
You need to check the whole project 

npx eslint


Not just components

Reply:
all the structure with same pattern right?

Reply:
done